### PR TITLE
maintenance beauty

### PR DESCRIPTION
**Pull request by bigcat88**
the same as for `giphy` and other repos

### CODE REVIEW

File: .nextcloudignore [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+/AUTHORS.md
+/CHANGELOG.md
+/CODE_OF_CONDUCT.md
+/README.md
+/SECURITY.md
+/composer.*
+/node_modules
+/screenshots
+/src
+/vendor
```
Comment (julien-nc) [inline .nextcloudignore:26]: This is dangerous IMO. Some repos don't use Mozart or any fancy thing to move dependencies in `lib/` so we might just wanna ignore `vendor/bin`.

Comment (bigcat88) [reply .nextcloudignore:26]: As we use almost the same file in all integrations, your notice is important.
Should  we change string from `/vendor` to `/vendor/bin`
if I understand correctly?

Comment (julien-nc) [reply .nextcloudignore:26]: Well if some Php packages are actually imported and used in the project, we should not exclude `vendor`.
So maybe yes, to be sure, let's never ignore `vendor` but always ignore `vendor/bin`.
As you wish:
* Just check this on future releases you make and adjust if needed
* Or change it everywhere if you feel like it :grin:

Comment (bigcat88) [reply .nextcloudignore]: changed here, in other let's change when something else will be added or adjusted to not create spam with one line of code 😁

**Pull request merged**